lib/utils.pm

```diff
@@ -803,6 +803,10 @@ sub power_action {
         $shutdown_timeout *= 3;
         record_soft_failure("boo#1057637 shutdown_timeout increased to $shutdown_timeout (s) expecting to complete.");
     }
+    # Override the timeout using a value from DEBUG_SHUTDOWN parameter.
+    if (my $debug_shutdown_timeout = get_var("DEBUG_SHUTDOWN")) {
+        $shutdown_timeout = $debug_shutdown_timeout;
+    }
```

Comment:
I see too much verbosity here. What do you think about:

```perl
$shutdown_timeout = get_var("DEBUG_SHUTDOWN") if (check_var('DEBUG_SHUTDOWN')) {
```

Reply:
I guess, it will not work as check_var (from API docs): 
```perl
check_var($variable, $value);
# Returns true if test variable $variable is equal to $value or returns undef
```

What do you think about the following:
```perl
$shutdown_timeout = get_var("DEBUG_SHUTDOWN") if (get_var('DEBUG_SHUTDOWN'));
```